nixos/doc/manual/development/non-switchable-systems.section.md

```diff
@@ -0,0 +1,21 @@
+# Non Switchable Systems {#sec-non-switchable-system}
+
+In certain systems, most notably image based appliances, updates are handled
+outside the system. This means that you do not need to re-build your
+configuration on the system itself anymore.
+
+If you want to build such a system, you can use the `image-based-appliance`
+profile:
+
+```nix
+{ modulesPath, ... }: {
+  imports = [ "${modulesPath}/profiles/image-based-appliance.nix" ]
+}
+```
+
+The most notable deviation of this profile from a standard NixOS system is that
+it disables your system from re-building itself. It sets
+`config.system.switch.enable = false;`, which excludes
+`switch-to-configuration`, the central script called by `nixos-rebuild`, from
+your system. Removing this script makes the image lighter and slightly more
+secure.
```

Comment:
This feels like an odd claim -- it doesn't prevent switching to another system, only from switching _to_ the system with this option enabled. Copying a "rebuildable" system into the Nix store on the machine in question, then running its switch-to-configuration, would still work as far as I understand...

Reply:
You're right of course. Changed the wording slightly. Should be more accurate now.